lib/tools/transformer/expression_generator.dart

```diff
@@ -84,8 +84,7 @@ class ExpressionGenerator extends Transformer with ResolverTransformer {
         .toList();
 
     // Get all of the contents of templates in @Component(templateUrl:'...')
-    gatherReferencedUris(transform, resolver, options,
-        templatesOnly: true).then((templates) {
```

Comment:
I would agree, but I am not sure on the motivation for having the option. I guess you are worried about parsing entities that aren't valid templates, such as css content. The problem is this isn't a good way of determining if they are template content. If you use ng_include then values in NgTemplateCache have html expressions.